emails/models.py

```diff
@@ -33,6 +34,51 @@ def __str__(self):
     def num_active_address(self):
         return RelayAddress.objects.filter(user=self.user).count()
 
+    def check_bounce_pause(self):
+        if self.last_hard_bounce:
+            last_hard_bounce_allowed = (
+                datetime.now(timezone.utc) -
+                timedelta(days=settings.HARD_BOUNCE_ALLOWED_DAYS)
+            )
+            if self.last_hard_bounce > last_hard_bounce_allowed:
+                return True, 'hard'
```

Comment:
This is fine but a named tuple might be cleaner for the code that's checking these answers.

Reply:
TIL. ❤️